yalexs_ble - Cipher not set

### The problem

Getting an unexpected error when attempting to add Yale YRL216-NR-619 Assure Level either manually via key/slot or when attempting to import from august integration. 



### What version of Home Assistant Core has the issue?

2023.1.5

### What was the last working version of Home Assistant Core?

_No response_

### What type of installation are you running?

Home Assistant OS

### Integration causing the issue

yalexs_ble

### Link to integration documentation on our website

https://www.home-assistant.io/integrations/yalexs_ble/

### Diagnostics information

_No response_

### Example YAML snippet

_No response_

### Anything in the logs that might be useful for us?

```txt
2023-01-18 13:22:58.364 ERROR (MainThread) [homeassistant.components.yalexs_ble.config_flow] Unexpected error
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/yalexs_ble/config_flow.py", line 187, in async_step_user
    await validate_lock(local_name, discovery_info.device, key, slot)
  File "/usr/src/homeassistant/homeassistant/components/yalexs_ble/config_flow.py", line 46, in validate_lock
    await PushLock(local_name, device.address, device, key, slot).validate()
  File "/usr/local/lib/python3.10/site-packages/yalexs_ble/push.py", line 479, in validate
    await self._update()
  File "/usr/local/lib/python3.10/site-packages/yalexs_ble/push.py", line 77, in _async_wrap_operation_lock
    return await func(self, *args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/yalexs_ble/push.py", line 98, in _async_wrap_retry_bluetooth_connection_error
    return await func(self, *args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/yalexs_ble/push.py", line 495, in _update
    state = await lock.status()
  File "/usr/local/lib/python3.10/site-packages/yalexs_ble/lock.py", line 298, in status
    response = await self._execute_command(
  File "/usr/local/lib/python3.10/site-packages/yalexs_ble/lock.py", line 271, in _execute_command
    response = await self.session.execute(
  File "/usr/local/lib/python3.10/site-packages/yalexs_ble/session.py", line 222, in execute
    return await write_task
  File "/usr/local/lib/python3.10/site-packages/yalexs_ble/session.py", line 105, in _write
    return await self._locked_write(command)
  File "/usr/local/lib/python3.10/site-packages/yalexs_ble/session.py", line 139, in _locked_write
    assert self.cipher_encrypt is not None, "Cipher not set"  # nosec
AssertionError: Cipher not set
```


### Additional information

_No response_

I'll also add this, in case it is not in the above mess :).

`Logger: yalexs_ble.push
Source: /usr/local/lib/python3.10/site-packages/yalexs_ble/push.py:744
First occurred: 2:17:02 PM (4 occurrences)
Last logged: 2:21:03 PM

Front door (AugSW38): Error updating
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/yalexs_ble/push.py", line 713, in _queue_update
    await self._update()
  File "/usr/local/lib/python3.10/site-packages/yalexs_ble/push.py", line 77, in _async_wrap_operation_lock
    return await func(self, *args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/yalexs_ble/push.py", line 98, in _async_wrap_retry_bluetooth_connection_error
    return await func(self, *args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/yalexs_ble/push.py", line 495, in _update
    state = await lock.status()
  File "/usr/local/lib/python3.10/site-packages/yalexs_ble/lock.py", line 298, in status
    response = await self._execute_command(
  File "/usr/local/lib/python3.10/site-packages/yalexs_ble/lock.py", line 271, in _execute_command
    response = await self.session.execute(
  File "/usr/local/lib/python3.10/site-packages/yalexs_ble/session.py", line 222, in execute
    return await write_task
  File "/usr/local/lib/python3.10/site-packages/yalexs_ble/session.py", line 105, in _write
    return await self._locked_write(command)
  File "/usr/local/lib/python3.10/site-packages/yalexs_ble/session.py", line 139, in _locked_write
    assert self.cipher_encrypt is not None, "Cipher not set"  # nosec
AssertionError: Cipher not set`

--- a/homeassistant/components/august/manifest.json
+++ b/homeassistant/components/august/manifest.json
@@ -2,7 +2,7 @@
   "domain": "august",
   "name": "August",
   "documentation": "https://www.home-assistant.io/integrations/august",
-  "requirements": ["yalexs==1.2.6", "yalexs_ble==1.12.5"],
+  "requirements": ["yalexs==1.2.6", "yalexs_ble==1.12.7"],
   "codeowners": ["@bdraco"],
   "dhcp": [
     {

--- a/homeassistant/components/yalexs_ble/manifest.json
+++ b/homeassistant/components/yalexs_ble/manifest.json
@@ -3,7 +3,7 @@
   "name": "Yale Access Bluetooth",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/yalexs_ble",
-  "requirements": ["yalexs-ble==1.12.5"],
+  "requirements": ["yalexs-ble==1.12.7"],
   "dependencies": ["bluetooth_adapters"],
   "codeowners": ["@bdraco"],
   "bluetooth": [

--- a/requirements_all.txt
+++ b/requirements_all.txt
@@ -2657,7 +2657,7 @@ yalexs-ble==1.12.7
 yalexs==1.2.6
 
 # homeassistant.components.august
-yalexs_ble==1.12.5
+yalexs_ble==1.12.7
 
 # homeassistant.components.yeelight
 yeelight==0.7.10

--- a/requirements_all.txt
+++ b/requirements_all.txt
@@ -2651,7 +2651,7 @@ xs1-api-client==3.0.0
 yalesmartalarmclient==0.3.9
 
 # homeassistant.components.yalexs_ble
-yalexs-ble==1.12.5
+yalexs-ble==1.12.7
 
 # homeassistant.components.august
 yalexs==1.2.6

--- a/requirements_test_all.txt
+++ b/requirements_test_all.txt
@@ -1879,7 +1879,7 @@ yalexs-ble==1.12.7
 yalexs==1.2.6
 
 # homeassistant.components.august
-yalexs_ble==1.12.5
+yalexs_ble==1.12.7
 
 # homeassistant.components.yeelight
 yeelight==0.7.10

--- a/requirements_test_all.txt
+++ b/requirements_test_all.txt
@@ -1873,7 +1873,7 @@ xmltodict==0.13.0
 yalesmartalarmclient==0.3.9
 
 # homeassistant.components.yalexs_ble
-yalexs-ble==1.12.5
+yalexs-ble==1.12.7
 
 # homeassistant.components.august
 yalexs==1.2.6

